### PR TITLE
[TASK] Add symfony service dependency injection container

### PR DESCRIPTION
**Pull request by sabbelasichon**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,11 @@
         "symfony/process": "^4.0",
         "symfony/options-resolver": "^4.0",
         "guzzlehttp/guzzle": "^6.0",
-        "webmozart/assert": "^1.4"
+        "webmozart/assert": "^1.4",
```
Comment (simonschaufi) [inline composer.json]: please sort them alphabetically (there is a composer setting for it which I set globally)

Comment (t3easy) [reply composer.json]: or we use ergebnis/composer-normalize to normalize the composer.json file

Comment (simonschaufi) [reply composer.json]: if not done globally, this is the local setting for it but I don't want to have another dependency added:
```
    "config": {
        "optimize-autoloader": true,
        "sort-packages": true
    },
```

File: src/Cli/Symfony/ConsoleApplication.php [hunk 2]
```diff
@@ -25,24 +23,20 @@ class ConsoleApplication extends Application
      */
     protected $factory;
 
-    public function setFactory(FactoryInterface $factory): void
+    /**
+     * @var OutputInterface
+     */
+    private $output;
+
+    public function __construct(FactoryInterface $factory, OutputInterface $output, string $name = 'TYPO3 Surf', string $version = '3.0.0-alpha')
```
Comment (simonschaufi) [inline src/Cli/Symfony/ConsoleApplication.php]: I dislike the version here. I would prefer setting this from the outside. Other opinions?

Comment (sabbelasichon) [reply src/Cli/Symfony/ConsoleApplication.php]: @simonschaufi I don´t like it either. But it has nothing to do with this PR. So let us focus on the real issue.

Comment (simonschaufi) [reply src/Cli/Symfony/ConsoleApplication.php]: ok. I thought you introduced the version number here as it was in another file before.

Comment (simonschaufi) [reply src/Cli/Symfony/ConsoleApplication.php]: @sabbelasichon I refactored it slightly. Are you ok with that?

**Pull request merged**